src/main-process/file-recovery-service.js

```diff
@@ -162,7 +162,7 @@ class RecoveryFile {
 
 async function tryStatFile(path) {
   return new Promise((resolve, reject) =>
-    fs.stat(path, (error, result) => resolve(error == null && result))
+    fs.stat(path, (error, result) => resolve(error ?? result))
```

Comment:
These two lines are not equivalent.

`error == null && result` = `error == null ? result : false`
`error ?? result` = `error == null ? result : error`

in the second one `error` is a truthy value.